pints/_nested/__init__.py

```diff
@@ -9,11 +9,191 @@
 from __future__ import absolute_import, division
 from __future__ import print_function, unicode_literals
 import pints
+import numpy as np
+from scipy.special import logsumexp
 
 
 class NestedSampler(pints.TunableMethod):
     """
     Abstract base class for nested samplers.
+    """
+    def __init__(self, log_prior):
+        # Store function
+        if not isinstance(log_prior, pints.LogPrior):
+            raise ValueError('Given log_prior must extend pints.LogPrior')
+        self._log_prior = log_prior
```

Comment:
Are these properties meant to be accessed directly by subclasses? In that case please add a one line comment above each

Reply:
Done!